volontulo/templates/organizations/organization_view.html

```diff
@@ -25,12 +25,16 @@
 
     <div class="form-horizontal">
         <div class="col-xs-offset-2 col-xs-10">
+            {% if allow_edit %}
             <a href="{% url 'organization_form' organization.name|slugify organization.id %}" class="btn btn-primary">Edytuj organizację</a>
```

Comment:
I don't know if it will work, becaues tip of master right now doesn't have organization_form view at all.